### PR TITLE
Change TLS RC4 cipher strength check to be data driven.

### PR DESCRIPTION
**Pull request by slontis**
This is a same pattern as used in PR #16652

##### Checklist

- [ ] documentation is added or updated
- [ ] tests are added or updated

### CODE REVIEW

File: ssl/s3_lib.c [hunk 1]
```diff
@@ -2823,7 +2823,7 @@ static SSL_CIPHER ssl3_ciphers[] = {
      0, 0,
      SSL_NOT_DEFAULT | SSL_MEDIUM,
```
Comment (paulidale) [inline ssl/s3_lib.c:2824]: MEDIUM or worse for these?

**Review by paulidale — APPROVED** [review]
Looks good, just the one question.

File: ssl/s3_lib.c (hunk 1 above)
Comment (slontis) [reply ssl/s3_lib.c:2824]: Not sure - nothing is at the low level currently. Note that this changes the sorting order when you modify this value - so I didnt want to mess with that here..

**Comment by slontis** [comment]
If PR #16652 is rejected then this should be also..

**Review by t8m — APPROVED** [review]
IMO this is right. Much better than hardcoding the conditions into the callback.

**Comment by t8m** [comment]
Merged to master. Thank you.

**Pull request closed without merging**